### PR TITLE
Removed 'host name' entry and updated all instances to hostname

### PR DESCRIPTION
**Pull request by bergerhoffer**
Addresses #108.

Removing "host name" entry and updating all instances of "host name" in the guide to "hostname".

Also updated the history file with the council's decision.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council for review please!

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc [hunk 1]
```diff
@@ -117,7 +117,7 @@ Do not use "Firewire" or "firewire." Although FireWire is a trademark of Apple C
 [discrete]
 [[fqdn]]
 ==== image:images/yes.png[yes] FQDN (noun)
-*Description*: "FQND" is an acronym for fully qualified domain name. A FQDN consists of a host and domain name, including top-level domain. For example, www.redhat.com is a fully qualified domain name. www is the host, redhat is the second-level domain, and .com is the top-level domain. A FQDN always starts with a host name and continues all the way up to the top-level domain name, so www.parc.xerox.com is also a FQDN.
+*Description*: "FQND" is an acronym for fully qualified domain name. A FQDN consists of a host and domain name, including top-level domain. For example, www.redhat.com is a fully qualified domain name. www is the host, redhat is the second-level domain, and .com is the top-level domain. A FQDN always starts with a hostname and continues all the way up to the top-level domain name, so www.parc.xerox.com is also a FQDN.
```
Comment (inoxx03) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc:120]:
```suggestion
*Description*: "FQDN" is an acronym for fully qualified domain name. A FQDN consists of a host and domain name, including top-level domain. For example, www.redhat.com is a fully qualified domain name. www is the host, redhat is the second-level domain, and .com is the top-level domain. A FQDN always starts with a hostname and continues all the way up to the top-level domain name, so www.parc.xerox.com is also a FQDN.
```

Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc]: Done :)

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council Can I get some reviews/approvals on this PR please?

**Review by mportman12 — APPROVED** [review]
LGTM

**Review by sterobin — APPROVED** [review]

**Review by inoxx03 — APPROVED** [review]
Thank you for fixing the FQDN acronym, LGTM, merging.

**Pull request merged**